src/lib/device/virtual_dev.hpp

```diff
@@ -92,7 +92,7 @@ class VirtualDev {
 
     std::mutex m_mgmt_mutex;          // Any mutex taken for management operations (like adding/removing chunks).
     std::set< PhysicalDev* > m_pdevs; // PDevs this vdev is working on
-    sisl::sparse_vector< shared< Chunk > > m_all_chunks; // All chunks part of this vdev
+    std::vector< shared< Chunk > > m_all_chunks; // All chunks part of this vdev
```

Comment:
Why is it replaced with std::vector<>? Chunks could be read in any order right and we wanted to place them in chunk_id order right?

Reply:
Reverted.